metricbeat/module/golang/heap/_meta/data.json

```diff
@@ -0,0 +1,46 @@
+{
+    "@timestamp": "2017-02-17T01:41:24.646Z",
+    "beat": {
+        "hostname": "MacBook-Pro.local",
+        "name": "MacBook-Pro.local",
+        "version": "6.0.0-alpha1"
+    },
+    "golang": {
+        "heap": {
+            "allocations": {
+                "active": 3325952,
+                "allocated": 2165152,
+                "total": 3393472,
+                "frees": 7975,
+                "idle": 1163264,
+                "mallocs": 27041,
+                "objects": 19066
+            },
+            "cmdline": "./metricbeat -httpprof=127.0.0.1:6060 -e -v",
+            "gc": {
+                "total_count": 1,
+                "total_pause_ns": 152192,
+                "next_gc_limit": 4194304,
+                "pause_ns": {
```

Comment:
Yes, that is what I mean. It is for naming purpose. As mentioned earlier, `max.ns` will break compatibility with 2.x releases. We initially used `max.ns` but had to change it because it started to break things. You are right that the code is not the nicest. Perhaps we find a nicer solution in the long run, but that is the current way we handle it.